vue.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  css: {
+    loaderOptions: {
+      sass: {
+        data: `
+            @import "@/assets/tokens/tokens.scss";
+            @import "@/assets/tokens/tokens.map.scss";
+            @import "@/styles/styles.scss";
```

Comment:
This is where tokens and helpers get loaded globally? If yes, ignore my other comment regarding webpackConfig

Reply:
it is "documented" [here](https://github.com/vuejs/vue-loader/issues/328)

Reply:
More specifically [here](https://cli.vuejs.org/guide/css.html#passing-options-to-pre-processor-loaders)

Reply:
Found it earlier too, thanks! :)